pyvista/plotting/axes_actor.py

```diff
@@ -0,0 +1,52 @@
+"""Axes actor module."""
+import pyvista as pv
+
+
+class AxesActor(pv._vtk.vtkAxesActor):
+    """Axes actor wrapper for vtkAxesActor.
+
+    Hybrid 2D/3D actor used to represent 3D axes in a scene. The user
+    can define the geometry to use for the shaft or the tip, and the
+    user can set the text for the three axes. To see full customization
+    options, refer to `vtkAxesActor Details
+    <https://vtk.org/doc/nightly/html/classvtkAxesActor.html#details>`
+
+    Examples
+    --------
+    Customize the axis shaft color and shape.
+
+    >>> import pyvista as pv
+    >>> axes = pv.Axes()
+    >>> axes.axes_actor.GetZAxisShaftProperty().SetColor(0, 1, 1)
+    >>> axes.axes_actor.SetShaftTypeToCylinder()
```

Comment:
Let's expose these methods on this class

Reply:
If you're up for exposing the property as well, great but not totally necessary IMO

Reply:
I exposed the relevant methods of the class. If we want to expose the properties as well, I think we should do it in another PR, since this is getting too big already.

Take a look and let me know what do you think 😄

Reply:
Great! Thanks! I will give this another pass soon here

Reply:
> If we want to expose the properties as well, I think we should do it in another PR

Laughs in 2000 LOC PRs.

I'll cover this. It will be less work in this PR.